spec/classes/profile/web_spec.rb

```diff
@@ -149,6 +149,34 @@
                   },
                   tag: ['st2', 'st2::backend', 'st2::backend::api'])
         end
+        it do
+          is_expected.to contain_nginx__resource__location('@basic_statusError')
```

Comment:
Should these tests only run when the basic status page is enabled?

Reply:
Yea working on fixing the tests. I have a pretty busy day today though so it might have to wait till tomorrow.